packages/InlineEditing/stories/InlineEditing.stories.mdx

```diff
@@ -0,0 +1,278 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import Blocks, { Block } from "storybook/components/Blocks";
+import { getStoryName } from "storybook/storyTree";
+import { AppInlineInput } from "./examples/InlineInput.simple.js";
+import TextAreaStory from "./examples/Textarea.js";
+import InputStory from "./examples/Input.js";
+import TableStory from "./examples/Table.js";
+import Toast from "../../Toast/src";
+
+import { ListBoxSingleStory, ListBoxMultipleStory } from "./examples/ListBox.js";
+
+export const params = {
+  options: {
+    isToolshown: true,
+    showPanel: false,
+  },
+};
+
+<Meta title={`${getStoryName("InlineEditing")}/Docs`} parameters={params} />
+
+<Toast kind={Toast.types.kind.WARNING} hasCloseButton={false}>
+  Work in progress ahead, this component still requires some more work to be ready for production, but the API is
+  stable, so you can start integrating it into your application.
+</Toast>
+
+# Inline Editing
+
+The `<InlineEditing />` component allows you to display two values to the user: one when the component is not in editing mode and an input whenever the user interact and set
```

Comment:
From what I can see, there is NOT an `<InlineEditing>` component, it's just an `InlineEditing` package with an `<Editor>` and a `<Table>` component...? (and maybe the other inline editable inputs _should_ also be exported?)

Reply:
Yeap indeed is a package with multiple components:

- Editor
- Table
- ListBox
- Input
- Textarea

I should reword that. 

> (and maybe the other inline editable inputs should also be exported?) 

The are exported as individual packages via `@paprika/inlinline-editing/lib/ListBox`

those are independents components, is interesting maybe there is not even a reason for an index in this set of components.